ui/pages/reservation-unit/[id].tsx

```diff
@@ -569,6 +569,14 @@ const ReservationUnit = ({
     );
   }, [reservationUnit.canApplyFreeOfCharge, reservationUnit.pricings]);
 
+  const isTouchDevice = useMemo(
```

Comment:
Could this be a utility function? then there would possibly be no need for shouldCalendarControlsBeVisible? could just use the same function?